### PR TITLE
Add some sentences about -webkit-line-clamp standard status

### LINKED ISSUE #30842: Information about how to use -webkit-line-clamp independent of -webkit-box-orient, and its future
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

### What specific section or headline is this issue about?

-webkit-line-clamp

### What information was incorrect, unhelpful, or incomplete?

There is no warning about `-webkit-line-clamp` being deprecated but in order to use it you must also use `-webkit-box-orient`, which is described as "[Non-standard and deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient)". What is the future of `-webkit-line-clamp`?

### What did you expect to see?

Information about the roadmap of `-webkit-line-clamp` and how it might be used independently of `-webkit-box-orient`, which is deprecated.

### Do you have any supporting links, references, or citations?

- https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient
- https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

### Do you have anything more you want to share?

I looked on caniuse.com and it has `box-orient` listed as usable only with the `-moz-` prefix but it also seems to work with the `-webkit-` prefix in Firefox.

https://caniuse.com/?search=box%20orient

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/-webkit-line-clamp`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/-webkit-line-clamp/index.md
* Last commit: https://github.com/mdn/content/commit/44cf523714745d626317192bfbe849b47144f3ab
* Document last modified: 2023-11-10T13:26:54.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
Fix https://github.com/mdn/content/issues/30842. @tannermann does this look good to you?

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/css/-webkit-line-clamp/index.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ browser-compat: css.properties.line-clamp
...
 
 When applied to anchor elements, the truncating can happen in the middle of the text, not necessarily at the end.
 
-> **Note:** This property was originally implemented in WebKit and has some issues. It got standardized in [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef--webkit-line-clamp) for legacy support. The [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef-line-clamp) specification also defines a {{cssxref("line-clamp")}} property, which is meant to replace this property and avoid its issues.
+> **Note:** This property was originally implemented in WebKit and has some issues, such as the dependency on two other legacy properties. It got standardized in [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef--webkit-line-clamp) for legacy support. The [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef-line-clamp) specification also defines a {{cssxref("line-clamp")}} property, which is meant to replace this property and avoid its issues. However, all browsers that support `line-clamp` will also support `-webkit-line-clamp` for compatibility reasons.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/-webkit-line-clamp/index.md:18]:
```suggestion
> **Note:** This property was originally implemented in WebKit and has some issues, such as the dependency on two other legacy properties. It got standardized in [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef--webkit-line-clamp) for legacy support. CSS Overflow Module Level 4 also defines a {{cssxref("line-clamp")}} property, which is meant to replace this property and avoid its issues. However, all browsers that support `line-clamp` will also support `-webkit-line-clamp` for compatibility reasons.
```

Comment (chrisdavidmills) [reply files/en-us/web/css/-webkit-line-clamp/index.md:18]: Minor nit really, but it felt a bit unnecessary to include two links to the same spec in one sentence.

**Review by chrisdavidmills — APPROVED** [review]
@Josh-Cena approving, but one minor nit for you to consider.

**Review by tannermann — COMMENTED** [review]
LGTM

**Pull request merged**